Assets/Hellmade/Eazy Sound Manager/Scripts/Audio.cs

```diff
@@ -434,7 +434,7 @@ public void Play(float volume)
             }
 
             // Recreate audiosource if it does not exist
-            if (AudioSource == null)
+            if (AudioSource == null || !AudioSource.enabled)
```

Comment:
Forgot to mention. This only happens when I spam the sound a fair amount. Let's say the player jumps 10-20 times a second in a small area, hitting his head a lot.

Reply:
Thanks for the fix. This should be merged in.